src/endpoint/iam/iam_rest.js

```diff
@@ -63,6 +63,8 @@ const ACTIONS = Object.freeze({
     'ListUserPolicies': 'list_user_policies',
     'ListUserTags': 'list_user_tags',
     'ListVirtualMFADevices': 'list_virtual_mfa_devices',
+    'TagUser': 'tag_user',
+    'UntagUser': 'untag_user',
```

Comment:
Could you move the tags operations after access keys (and also move the list user tags near)?